src/views/HomePageViews/HeroSection/HeroSection.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Marquee from 'react-fast-marquee';
+import {
+  StyledHeroSection,
+  Dates,
+  StarSeparator,
+  MapPin,
+  AddressFirstLine,
+  AddressSecondLine,
+} from './HeroSection.styles';
+import Center from '../../../components/LayoutSections/Center';
+import starSeparator from '../../../images/star-separator.svg';
+import mapPin from '../../../images/map-pin.svg';
+
+const HeroSection = () => {
+  return (
+    <StyledHeroSection>
+      <Marquee gradient={false} speed={75}>
+        <StarSeparator src={starSeparator} alt='' />
```

Comment:
Ajouter l'attribut `role=presentation` pour enlever le sens sémantique à toute image qui ne sert qu'à titre illustratif (accessibilité) ([en savoir plus](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role))

Reply:
J'veux pas m'obstiner, mais selon la doc de WAI-ARIA, un alt vide a le même effet: https://www.w3.org/TR/wai-aria-practices-1.1/#presentation_role
"Hiding a decorative image; it is equivalent to giving the image null alt text."

Reply:
@KrystinaH Par définition, `alt=''` indique à un lecteur d'écran de skipper l'image qui est purement décorative. `role=presentation` est un attribut ARIA qui peut être utlisé pour masquer des éléments à des technos d'assistance tel que les lecteurs d'écrans ET autres. Pour une couverture supplémentaire, on peut donc utliser les 2 en même temps.

https://www.portent.com/blog/design-dev/decorative-images-how-to-identify-and-tag-them-for-accessibility.htm